### ISSUE TITLE
Resource::get_rid is not exposed as virtual

### ISSUE THREAD

**Issue by mohsenph69** [godotengine/godot-cpp#1640]
### Godot version

4.3

### godot-cpp version

4.3

### System information

Linux manjaro

### Issue description

Resource::get_rid is not exposed as virtual, and that make it impossible to create a custom mesh!

### Steps to reproduce

NOTHING

### Minimal reproduction project

NOTHING

**Comment by dsnopek** [comment]
The virtual function is `Resource::_get_rid()` (with an underscore)

**Comment by mohsenph69** [comment]
> The virtual function is `Resource::_get_rid()` (with an underscore)

Yeah with an underscore, it exist in GDScript but not GDExtension!
